src/client/components/ApplicationSidebar.jsx

```diff
@@ -11,10 +11,11 @@ import { Switch } from "./ui/Switch";
 import { Label } from "./ui/Label";
 import { saveScene } from "../store/scene";
 import { useDispatch, useSelector } from "react-redux"
+import { setAction } from "../store/currentAction";
 
 function ApplicationSidebar({ className }) {
   const dispatch = useDispatch()
-  const scene = useSelector(state => state.scene)
+  const state = useSelector(state => state)
```

Comment:
I don't like this change,  this is grabbing the entirety the application state.

We should be keeping the editor to its own slice like .scene or .editor

Reply:
if you only need a few, you could destructure;
`const {scene, currentAction,selectedPoint}  = useSelector(state => state)`